cr.sh

```diff
@@ -239,7 +237,7 @@ update_index() {
     git add index.yaml
     git commit --message="Update index.yaml" --signoff
 
-    local repo_url="https://x-access-token:$CR_TOKEN@github.com/$owner/$repo"
+    local repo_url="https://x-access-token:$GITHUB_TOKEN@github.com/$owner/$repo"
```

Comment:
But the variable is still named CR_TOKEN?

Reply:
We should revert that and use CR_TOKEN here

Reply:
just did that